src/Domain/WebSockets/WebSocketsService.ts

```diff
@@ -0,0 +1,52 @@
+import { RoleName } from '@standardnotes/auth'
+import { AxiosInstance } from 'axios'
+import { inject, injectable } from 'inversify'
+
+import TYPES from '../../Bootstrap/Types'
+import { DomainEventFactoryInterface } from '../Event/DomainEventFactoryInterface'
+import { User } from '../User/User'
+import { WebSocketsConnectionRepositoryInterface } from './WebSocketsConnectionRepositoryInterface'
+import { WebSocketsServiceInterface } from './WebSocketsServiceInterface'
+
+@injectable()
+export class WebSocketsService implements WebSocketsServiceInterface {
```

Comment:
Goot stuff 👍 

Hmm what do you think about the following change:
- rename the `src/Domain/WebSockets/WebSocketsServiceInterface` to `src/Domain/Client/ClientServiceInterface`
- rename the `src/Domain/WebSockets/WebSocketsService` to `src/Infra/WebSockets/WebSocketsClientServiceInterface` 

?

This would impose a clear separation between the domain (a service for communicating with the clients) and the implementation detail - which is websockets as an infra technology used for communicating the clients.

Reply:
Agreed, would be similar to how MySQL and Redis repos live in Infra and their interfaces live in Domain 👍